grafana/RELEASES.md

```diff
@@ -5,9 +5,9 @@
 - Support Grafana 4.4.1.
 
 ## 4.0.2 (04-01-2017)
-- Formalized the image name for every arch to `gcr.io/google_containers/heapster-grafana-ARCH:VERSION`
+- Formalized the image name for every arch to `k8s.gcr.io/heapster-grafana-ARCH:VERSION`
```

Comment:
Substituting this in old release notes doesn't seem to be the right thing to do.